lxqt-config-appearance/gtkconfig.cpp

```diff
@@ -77,8 +81,13 @@ void GTKConfig::applyGTKStyle()
 {
     if (ui->advancedOptionsGroupBox->isChecked())
     {
+        // GTK4
+        QString themeName = ui->gtk4ComboBox->currentText();
+        mConfigOtherToolKits->setGTKConfig(QStringLiteral("4.0"), themeName);
+        if(QGuiApplication::platformName() == QStringLiteral("wayland"))
+            mConfigOtherToolKits->setGsettingsConfig(QStringLiteral("4.0"), themeName);
         // GTK3
```

Comment:
The call to `gsettings` (in `setGsettingsConfig`) will be repeated and overridden below, for GTK3.

I think GTK3 and GTK4 should have the same settings and be set by a single call to `gsettings`. If I'm right, a single combo-box should be used for both, the previous commit should be reverted, and some minor changes will be needed too.

What I say is based on what I see in `dconf-editor`: when I set it to `HighContrast`, both GTK3 and GTK4 themes change. The call to `gsettings` does the same.

This creates a confusing situation though: what if a theme doesn't support GTK4? I think, in that case, only the GTK3 theme will be applied.